src/sentry/static/sentry/app/components/pageHeader.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+
+class PageHeader extends React.Component {
```

Comment:
Nomenclature nit: Shouldn't this be `PageHeading` instead of `PageHeader` since we're styling `h1` and `h4`'s and not `divs` or `Headers`?

Reply:
Yes. Good point.

Reply:
done!